src/assets/i18n/it.json5

```diff
@@ -7838,5 +7852,119 @@
   // "admin.system-wide-alert.title": "System-wide Alerts",
   "admin.system-wide-alert.title": "Allarmi di sistema",
 
+  // "external-login.confirmation.header": "Information needed to complete the login process",
```

Comment:
All the i18n keys starting from here & below have only been added to `it.json5`.  They are missing from `en.json5`.  They need to be copied or moved over to `en.json5`, otherwise the interface will not work in English.